### PR TITLE
redis-cli TLS connections

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/DOC-1760/rs/references/cli-utilities/redis-cli/#connect-over-tls)

### CODE REVIEW

**Review by yoavredis — CHANGES_REQUESTED** [review]
Looks OK to me.
But I think it should also be reviewed and approved by those who maintain redis-cli.
I suggest to ask @oranagra to take a look

File: content/rs/references/cli-utilities/redis-cli/_index.md [hunk 1]
```diff
@@ -107,6 +132,26 @@ OK
...
 ```
 
+## Examples
+
+### Check slowlog
+
+Run [`slowlog get`](https://redis.io/commands/slowlog-get/) for a list of recent slow commands:
+
+```sh
+redis-cli -h <endpoint> -p <port> slowlog get <number of entries> with-complexity
```
Comment (oranagra) [inline content/rs/references/cli-utilities/redis-cli/_index.md:142]: i suggest dropping the `with-complexity` part,
it's not an official feature

**Review by oranagra — COMMENTED** [review]
i've never connected with redis-cli to RE using TLS, so i can't comment on most of that text.
i suggest asking a proxy developer or QA person that tested it.

**Review by kaitlynmichael — APPROVED** [review]
One wording suggestion

**Pull request merged**